content/client-lib-development-guide/features.textile

```diff
@@ -1525,6 +1561,7 @@ class Rest:
   constructor(ClientOptions) // RSC1
   auth: Auth // RSC5
   push: Push
+  batch: BatchOperations // BO1
```

Comment:
I agree, this didn't feel like a 'perfect' solution to me either. I was stuck for a better way to do it though, especially for batch publishing. I could add batch presence to RestPresence easily, but batch publishing to multiple channels didn't feel like it would work in the same way as publishing happens on a Channel object

Reply:
The only alternative I can think of is to have `RestChannels#batchPublish` and `RestPresence#batchGet`, @paddybyers WDYT?